docs/notebooks/tutorial_axes_anndata_mudata.ipynb

```diff
@@ -0,0 +1,1084 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def highlyvar(adata, flavor: str, ngenes= None, min_mean=0.02, max_mean=4, min_disp=0.5 ):
</pre><p>is this helper function necessary? seurat v3 doesn't seem to be used...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>